tilelang/backend/target.py

```
@@ -0,0 +1,307 @@
+        if self.host is not None:
+            config["host"] = _target_input_to_tvm(self.host)
+        return config
+
+    def to_tvm_target(self) -> TVMTarget:
+        target_input = self.to_tvm_target_input()
+        target = target_input if isinstance(target_input, TVMTarget) else TVMTarget(target_input)
+        try:
+            from tilelang.utils.target import with_rocm_target_attrs
+
+            return with_rocm_target_attrs(target)
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)